models/optional_modules/app_errors/scratch/snowflake/snowplow_mobile_app_errors_this_run.sql

```diff
@@ -0,0 +1,89 @@
+{{ 
+  config(
+    cluster_by = ["derived_tstamp"],
```

Comment:
Up until now I have avoided clustering the `this_run` tables for Snowflake, and only clustered the derived tables. There is a cost associated to reclustering in Snowflake + clustering is best suited to very large table, which the this run tables are not really. I am not too precious about this though